sirepo/auth/__init__.py

```diff
@@ -495,19 +501,21 @@ def user_name():
     )
 
 
-def user_registration(uid):
+def user_registration(uid, display_name=None):
```

Comment:
Remove =None

Reply:
Why? In the non-pkcli case we don't have a display name when this is called. Should I just pass in None in that case?

Reply:
I was confused. I thought this was only for the pkcli. That's fine. leave as is.